src/games/snake/snake_part.py

```diff
@@ -0,0 +1,24 @@
+import pygame
+from src.resources.snake_game_resources import SnakeGameResources
+
+
+class SnakePart(pygame.sprite.Sprite):
+    def __init__(self, belongs_to, board, pos_center = None):
```

Comment:
belongs_to  was pretty confusing for me. I thought that it's boolean or something weird. What do you think about changing it to parent, parent_snake or anything similar?